src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateProbationBookingServiceTest.kt

```diff
@@ -125,7 +127,7 @@ class CreateProbationBookingServiceTest {
       .hasBookingType(BookingType.PROBATION)
       .hasProbationTeam(requestedProbationTeam)
       .hasMeetingType(ProbationMeetingType.PSR)
-      .hasComments("probation booking comments")
+      .hasStaffNotes("probation notes for staff")
       .hasCreatedBy(PROBATION_USER)
```

Comment:
Mixed phrases - `hasStaffNotes` and `notesForStaff` ... clearer if consistent?  i.e. `hasNotesForStaff`?